deps/src/main/java/com/microsoft/azure/sdk/iot/deps/serializer/ManagedIdentity.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.sdk.iot.deps.serializer;
+
+import lombok.Getter;
+import lombok.Setter;
+
+public class ManagedIdentity {
+    /**
+     * The managed identity used to access the storage account for IoT hub import and export jobs.
+     * For more information, see TODO
```

Comment:
Assuming you will be updating this later.

Reply:
yes have a task for it. But good point. I think I'll remove this for now though and then update later as they don't have it ready yet.